SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -29,6 +29,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bUseFrameTimeAsLoad(false)
 	, bCheckRPCOrder(false)
 	, bBatchSpatialPositionUpdates(true)
+	, MaxDynamicallyAttachedSubobjectsPerClass(3)
```

Comment:
What's the significance of '3'

Reply:
Also, if this changes don't we need to regen schema? We should probably make a notificaiton if this changes or something.